backend/copr_backend/job.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, task_data, worker_opts):
         self.results = None
         self.appstream = None
 
+        self.priority = None
```

Comment:
I would probably call this "repo_priority", so we don't intuitively mix-up with the task priority we have all around the code.